docs/diagnostics/PX1077.md

```diff
@@ -0,0 +1,86 @@
+# PX1077
+This document describes the PX1077 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                  | Type    | Code Fix    | 
+| ------ | ---------------------------------------------------------------------------------- | ------- | ----------- | 
+| PX1077 | Event handlers should have `protected` accessibility level and `virtual` modifier. | Warning | Available   | 
```

Comment:
The summary should be the same as in the summary page after the changes

Reply:
Done.